charts/metallb/values.yaml

```diff
@@ -127,6 +127,8 @@ prometheus:
 # controller.
 controller:
   enabled: true
+  # -- Controller log level. Must be one of: `all`, `debug`, `info`, `warn`, `error` or `none`
+  logLevel: info
```

Comment:
How about changing the manifests in `manifests/` to match the helm chart default value of `info` ?

Reply:
Makes sense, added.